django_app/redbox_app/redbox_core/admin.py

```diff
@@ -4,14 +4,25 @@
 
 
 class UserResource(admin.ModelAdmin):
-    exclude = ["password"]
+    fields = ["email", "is_superuser", "is_staff"]
 
 
 class FileResource(admin.ModelAdmin):
-    list_display = ["original_file_name", "user", "processing_status", "original_file", "core_file_uuid"]
+    list_display = ["original_file_name", "user", "processing_status"]
+
+
+
+class ChatMessageInline(admin.StackedInline):
```

Comment:
this is a little neater than putting the individual messages in their own admin

Reply:
We need to remember that any admins will need OS clearance when allowing any external admins